go/src/lib.rs

```diff
@@ -1,9 +1,159 @@
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
please move this after license header (hint: you need to start it with `/*` isntead of `/**`)